raiden/message_handler.py

```diff
@@ -247,7 +247,7 @@ def handle_message_lockexpired(
         balance_proof = balanceproof_from_envelope(message)
         lock_expired = ReceiveLockExpired(
             sender=balance_proof.sender,
-            balance_proof=balance_proof,
+            balance_proof=balance_proof,  # type: ignore
```

Comment:
What is the error you're getting without this ignore?

Reply:
I just get `raiden/message_handler.py:297: error: Unexpected keyword argument "balance_proof" for "ReceiveTransferRefund"`

Reply:
From what I can see, the CI looks right and your error looks wrong. Do you agree?

You could try to clear the mypy cache (`rm -fr .mypy_cache`) and check that your mypy has the right version.

Reply:
I agree, but the CI also has `error: Unexpected keyword argument "balance_proof"`

Reply:
that shouldnt be the case should it?

Reply:
```
Mypy : 0.782
mypy-extensions: 0.4.3
```

Reply:
That error shouldn't be there. This can be avoided by https://github.com/raiden-network/raiden/commit/d9d8c0e9841e4cf090940326a0b8e354fc0bb458. But that's not a solution and I don't know why that helps, yet.